README.md

```diff
@@ -177,94 +104,14 @@ The following variables can prepend the make target below
 * BUILD_TOOL, default docker
 * OLM_TYPE, default integreatly-operator
 
-### Deployment Scenarios
-
-Depending on the deployment type, you need to specify whether it is RHOAM or RHMI that you would like to work with.
-
-For RHMI:
-```sh
-export OLM_TYPE=integreatly-operator
-```
-For RHOAM:
-```sh
-export OLM_TYPE=managed-api-service
-```
-
-#### Deploy the latest bundle version 
-This assumes no prior installation of the RHMI/RHOAM operator. As such, it will remove the replaces field in the CSV file to 
-prevent the attempted replacement of an existing version of the operator available for installation.
-
-```sh
-ORG=<YOUR_ORG> make install/olm/bundle 
-```
-
-#### Deploy a specific bundle version
-This will assume not upgrading and remove the replaces field from the CSV
-
-Note: The version you are pointing to, must exist in the deploy/OLM_TYPE 
-
-```sh
-ORG=<YOUR_ORG> BUNDLE_VERSIONS="1.1.0" make install/olm/bundle
-```
-
-#### Deploy an upgrade version
-Example shows upgrade from 1.1.0 to 1.2.0. 1.1.0 must already be installed on the cluster and a 1.2.0 release must be available.
-1.2.0 must reference 1.1.0 in the CSV replaces field but 1.1.0 can not have the "replaces" field set.
-
-```sh
-ORG=<YOUR_ORG> BUNDLE_VERSIONS="1.2.0,1.1.0" UPGRADE=true make install/olm/bundle 
-```
-
-#### Create a new bundle and deploy
-The SEMVER version should be a logical SEMVER increment of the existing lastest bundle. 
-The latest bundle will be copied and used as a reference for this new release.  
-**NOTE:** If creating a new operator image in your repository, you need to update the CSV with references
-to your image.
-```sh
-ORG=<YOUR_ORG> SEMVER=<X.Y.Z> make release/prepare
-ORG=<YOUR_ORG> TAG=<X.Y.Z> make image/build/push    (create a new operator image if required)
-ORG=<YOUR_ORG> make install/olm/bundle 
-```
-**NOTE:** If creating a new version to replace an existing i.e. upgrade, add both versions to the BUNDLE_VERSIONS VARIABLE
-```sh
-ORG=<YOUR_ORG> BUNDLE_VERSIONS="<NEW-SEMVER>,<OLD-SEMVER>" make install/olm/bundle 
-```
-
 
+Run the script  ./scripts/bundle-rhmi-opertors.sh
 
 ### Install from OperatorHub
 OLM will created a PackageManifest (integreatly) based on the CatalogSource (rhmi-operators) in the openshift-marketplace namespace. 
 Confirm both and then find the RHMI in the OperatorHub. Verify that the version references the latest version available in the index and click install
 
 
-## Deploying to a Cluster with OLM and OperatorSource (Deprecated)
```

Comment:
This is marked as deprecated. It's probably had enough time as deprecated to be removed.